src/lib.rs

```diff
@@ -1514,7 +1514,11 @@ pub extern "C" fn zendoo_finalize_ginger_mht_in_place(
 
     match finalize_ginger_mht_in_place(tree) {
         Ok(_) => true,
-        Err(_) => false
+        Err(_) => {
```

Comment:
You missed the "e" e.g. Err(e). It will crash

Reply:
Done